src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -102,28 +102,30 @@ private static string GetNormalizedString(string input)
         private class FunctionActivityTracker : IDisposable
         {
             private readonly string executionId = Guid.NewGuid().ToString();
+            private readonly object functionMetricEventLockObject = new object();
             private DateTime startTime = DateTime.UtcNow;
             private ulong totalExecutionCount = 0;
-            private ulong runningFunctionCount = 0;
             private const int MetricEventIntervalInSeconds = 5;
             private CancellationTokenSource etwTaskcancellationSource = new CancellationTokenSource();
             private ConcurrentQueue<FunctionMetrics> functionMetricsQueue = new ConcurrentQueue<FunctionMetrics>();
-
+            private Dictionary<string, RunningFunctionInfo> runningFunctions = new Dictionary<string, RunningFunctionInfo>();
+            
             internal FunctionActivityTracker()
             {
                 Task.Run(
                     async () =>
                     {
                         try
                         {
-                            int currentSecond = MetricEventIntervalInSeconds;
+                            int currentSecond = 0;
                             while (!etwTaskcancellationSource.Token.IsCancellationRequested)
                             {
                                 RaiseMetricsPerFunctionEvent();
                                 currentSecond = currentSecond + 1;
                                 if (currentSecond >= MetricEventIntervalInSeconds)
                                 {
                                     RaiseMetricEtwEvent(ExecutionStage.InProgress);
```

Comment:
Yes, we still this ETW event till we update the function infrastructure with the new bits. This will be a staged deployment:
1. Function SDK bits will be updated first in prod.
2. Function infrastructure will be updated next.
3. Once function intra bits have been updated, I will remove the other EtwEvent.